stylesheets/_component.forms.scss

```diff
@@ -170,9 +170,21 @@ fieldset {
     }
 }
 
-.legend {
-    font-size: $font-size-large;
-    margin-bottom: .5em;
+fieldset {
```

Comment:
Feels like we should keep the removed `>` here. I can see some potential issues around nested fieldsets (dob fields)